test/units/testsuite-75.sh

```diff
@@ -389,6 +390,23 @@ grep -qF "myservice.signed.test:1234" "$RUN_OUT"
 grep -qF "10.0.0.20" "$RUN_OUT"
 grep -qF "fd00:dead:beef:cafe::17" "$RUN_OUT"
 grep -qF "authenticated: yes" "$RUN_OUT"
+
+# Test service resolve over Varlink
+(printf '
+{
+  "method": "io.systemd.Resolve.ResolveService",
+  "more": false,
+  "parameters":{"name":"","type":"_mysvc._tcp","domain":"signed.test"}
+}\0')| nc -U /run/systemd/resolve/io.systemd.Resolve > $VL_RESOLVE_LOGS
+
+grep -qF '"srv":{"priority":10,"weight":5,"port":1234,"hostname":"myservice.signed.test"}' "$VL_RESOLVE_LOGS"
+grep -qF '{"ifindex":4,"family":10,"address":[253,0,222,173,190,239,202,254,0,0,0,0,0,0,0,23]}' "$VL_RESOLVE_LOGS"
+grep -qF '{"ifindex":4,"family":2,"address":[10,0,0,20]}' "$VL_RESOLVE_LOGS"
+grep -qF '"normalized":{"normalized":"myservice.signed.test"}' "$VL_RESOLVE_LOGS"
```

Comment:
hmm a field which contains an object whith a field also called like the object#s field? that doesn't look right

Reply:
removed the wrapper field